apps/src/sites/studio/pages/levels/_level_group.js

```diff
@@ -165,9 +169,12 @@ function initLevelGroup(levelCount, currentPage, lastAttempt) {
         : i18n.submittableComplete();
     }
 
-    const confirmationDialog = (
-      <LegacySingleLevelGroupDialog id={id} title={title} body={body} />
-    );
+    let confirmationDialog = null;
+    if (!isActivityGuideLevel) {
+      confirmationDialog = (
+        <LegacySingleLevelGroupDialog id={id} title={title} body={body} />
+      );
+    }
```

Comment:
Not worth a change, since they both end up the same place and there's upsides/downsides to both, but an alternative way one might write this is as a one-liner:
```
const confirmationDialog = isActivityGuideLevel ? null : <LegacySingleLevelGroupDialog id={id} title={title} body={body} />
```

The downsides are a longer line. The upsides are it uses a const variable, which lets people skimming the code know this is the only place they need to think about it changing, and it avoids the very slightly harder to think about `if (!something)` clause.